google/cloud/firestore_v1/base_aggregation.py

```diff
@@ -264,9 +253,8 @@ def get(
                 explain_metrics will be available on the returned generator.
 
         Returns:
-            QueryResultsList[AggregationResult] | Awaitable[Any, Any, List[AggregationResult]]:
+            (QueryResultsList[List[AggregationResult]] | List[List[AggregationResult]]):
```

Comment:
Sounds good, but after trying it out I think both`Coroutine[Any, Any, List[List[AggregationResult]]]` and `Awaitable[List[List[AggregationResult]]]` work? (maybe I'm missing something)

Reply:
This is what I see:

```
google/cloud/firestore_v1/async_aggregation.py:49: error: Signature of "get" incompatible with supertype "BaseAggregationQuery"  [override]
google/cloud/firestore_v1/async_aggregation.py:49: note:      Superclass:
google/cloud/firestore_v1/async_aggregation.py:49: note:          def get(self, transaction: Any = ..., retry: Retry | _MethodDefault | None = ..., timeout: float | None = ..., *, explain_options: ExplainOptions | None = ...) -> QueryResultsList[list[AggregationResult]] | Coroutine[Any, Any, list[AggregationResult]]
google/cloud/firestore_v1/async_aggregation.py:49: note:      Subclass:
google/cloud/firestore_v1/async_aggregation.py:49: note:          def get(self, transaction: Any = ..., retry: AsyncRetry | _MethodDefault | None = ..., timeout: float | None = ...) -> Coroutine[Any, Any, list[AggregationResult]]
```

But if you found it works for you, that's fine with me too! Either way, I think this can fall into the known-broken mypy issues to fix later

Reply:
I see, I will revert the change and save this for later haha